app/src/main/java/de/xikolo/controllers/main/MainActivity.kt

```diff
@@ -284,14 +280,9 @@ class MainActivity : ViewModelActivity<NavigationViewModel>(), NavigationView.On
             }
 
             viewModel.user?.let { user ->
-                val profile: Profile? = user.profile
-                headerView.findViewById<TextView>(R.id.textName).text = String.format(
-                    getString(R.string.user_name),
-                    profile?.firstName ?: "",
-                    profile?.lastName ?: ""
-                )
-
-                headerView.findViewById<TextView>(R.id.textEmail).text = profile?.email ?: ""
+                headerView.findViewById<TextView>(R.id.textName).text = user.name
+
+                headerView.findViewById<TextView>(R.id.textEmail).text = user.profile.email
```

Comment:
`user.profile` can be null in some cases. I added the null check once as a fix. Is there a particular reason you removed it?